write-rustdoc-hide-lines/write_rustdoc_hide_lines.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+cargo run --release -- ../content/learn/book
```

Comment:
I wonder if we should just run this in CI. I assume right now the intended workflow is to let authors run it manually

Reply:
Yes, for now the idea is to run it manually… but probably makes sense to do it on CI. Anyway, I haven't looked into it as I don't know almost nothing about setting up GitHub CI (and didn't want to research). :-)

Reply:
No worries, that can be done in a separate PR, I can take care of the CI stuff.